doc/source/server_architecture.rst

```diff
@@ -130,6 +133,9 @@ Here is an example of how the **server** portion of **config.yaml** should look:
                 outputs:
                     - command_testbed_stream
 
+        api-streams:
```

Comment:
@MJJoyce Should this be 'api-telemetry-streams:' ?

Reply:
oooof course I missed one ...